common/notifs/discord.go

```diff
@@ -0,0 +1,95 @@
+package notifs
+
+import (
+	"log"
+	"net/url"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/disgoorg/disgo/discord"
+	"github.com/disgoorg/disgo/rest"
+	"github.com/disgoorg/disgo/webhook"
+	"github.com/disgoorg/snowflake/v2"
+
+	"github.com/ceramicnetwork/go-cas/models"
+)
+
+type DiscordColor int
+
+const (
+	DiscordColor_None    = iota
+	DiscordColor_Info    = 3447003
+	DiscordColor_Ok      = 3581519
+	DiscordColor_Warning = 16776960
+	DiscordColor_Alert   = 16711712
+)
+
+const DiscordPacing = 2 * time.Second
+
+type DiscordHandler struct {
+	alertWebhook   webhook.Client
+	warningWebhook webhook.Client
+	testWebhook    webhook.Client
+}
+
+func NewDiscordHandler() (models.Notifier, error) {
```

Comment:
does this require all webhook urls to be set?

Reply:
No, it checks for them and leaves them null if it doesn't find them set in the env.

Reply:
AHHHH missed that sorry